src/reducers/ChangePassword.reducer.test.ts

```diff
@@ -0,0 +1,110 @@
+import { describe } from 'riteway'
+
+import { Actions } from '../actions'
+import { changePassword } from './ChangePassword.reducer'
+
+const { onChangePassword, onChangePasswordError, onChangePasswordSuccess } = Actions.ChangePassword
+const createState = (obj?: any) => ({ ...changePassword(), ...obj })
+
+describe('changePassword reducer', async (should: any) => {
+  const { assert } = should()
+
+  assert({
+    given: 'no arguments',
+    should: 'return the default state',
```

Comment:
This test doesn't test that the default state is returned like it should.

The reason is that `createState` uses `changePassword` to get the default state, therefor changePassword() and createState() will always return the same thing, even if `changePassword` does not correctly return the default state.

I would import the defaultState from the reducer, then adjust `createState` to use defaultState instead of `changePassword`